README.md

```diff
@@ -27,43 +27,94 @@ If you don't have `conda` available, we recommend installing [Miniconda for Pyth
 Building from source
 --------------------
 
-This plugin uses [CMake](https://cmake.org/) as its build system.  
-Before compiling you must install [LibTorch](https://pytorch.org/cppdocs/installing.html), which is the PyTorch C++ API, by following the instructions at https://pytorch.org.
-You can then follow these steps:
+### Prerequisites
 
-1. Create a directory in which to build the plugin.
+- Minconda https://docs.conda.io/en/latest/miniconda.html#linux-installers
+- CUDA Toolkit https://developer.nvidia.com/cuda-downloads 
+- LibTorch which is the PyTorch C++ API, by following the instructions at https://pytorch.org/cppdocs/installing.html and https://pytorch.org. 
+    - to get a version for CUDA 1.16 used in our build example we use the commands:
+    ```
+        wget https://download.pytorch.org/libtorch/cu116/libtorch-cxx11-abi-shared-with-deps-1.13.0%2Bcu116.zip
+        unzip libtorch-cxx11-abi-shared-with-deps-1.13.0%2Bcu116.zip
+    ```
 
-2. Run the CMake GUI or `ccmake`, specifying your new directory as the build directory and the top
-level directory of this project as the source directory.
+### Build & install
 
-3. Press "Configure".  (Do not worry if it produces an error message about not being able to find PyTorch.)
+1. Get the source code
 
-4. Set `OPENMM_DIR` to point to the directory where OpenMM is installed.  This is needed to locate
-the OpenMM header files and libraries.  If you are unsure of what directory this is, the following
-script will print it out.
+   ```
+   git clone https://github.com/openmm/openmm-torch.git
+   cd openmm-torch
+   ```
 
-```python
-from simtk import openmm
-import os
-print(os.path.dirname(openmm.version.openmm_library_path))
+2. Set CUDA_HOME  (you may have a different path and version)
+
+   ```
+   export CUDA_HOME=/usr/local/cuda-11.6
+   ```
+
+3. Create and activate a conda environment using the provided environment file
+
+
+   ```
+   conda env create -n openmm-torch -f environment.yaml
+   conda activate openmm-torch
+   ```
+
+4. Configure 
+   ```
+   mkdir build && cd build
+
+   cmake .. -DPYTORCH_DIR=<path/to/libtorch> \
```

Comment:
If you install the environment with `conda`, you already know what the paths are.